src/shogun/clustering/KMeansBase.cpp

```diff
@@ -321,7 +321,7 @@ void KMeansBase::init()
 	SG_ADD(
 	    &use_kmeanspp, "kmeanspp", "Whether to use kmeans++",
 	    ParameterProperties::HYPER | ParameterProperties::SETTING);
-	SG_ADD(&mus, "mus", "Cluster centers", ParameterProperties::MODEL);
+	SG_ADD(&cluster_centers, "cluster_centers", "Cluster centers", ParameterProperties::MODEL);
```

Comment:
as @gf712 suggested this should be READONLY.
and there should be another parameter `initial_cluster_centers` that actually holds the cluster centers that can be provided by the user